Makefile

```diff
@@ -71,6 +70,27 @@ dev_drop_database:
 dev_mock_oeci_up:
 	docker-compose -f docker-compose.dev.yml -f src/frontend/developerUtils/docker-compose.mock-oeci.yml up -d
 
+deploy: deploy_backend deploy_frontend
+
+.ONESHELL:
```

Comment:
what is ONESHELL?

Reply:
Each command in a make task will otherwise be run in a separate shell so `cd` becomes effectively useless.